src_c/math.c

```diff
@@ -4074,6 +4093,7 @@ math_disable_swizzling(pgVector *self, PyObject *_null)
 }
 
 static PyMethodDef _math_methods[] = {
+    {"clamp", math_clamp, METH_FASTCALL, DOC_PYGAMEMATHCLAMP},
```

Comment:
You'll have to also change this
```suggestion
#if PY_VERSION_HEX < 0x03070000
    {"clamp", math_clamp_wrapper, METH_VARARGS, DOC_PYGAMEMATHCLAMP},
#else 
    {"clamp", math_clamp, METH_FASTCALL, DOC_PYGAMEMATHCLAMP},
#endif    
```